config/samples/csi_v1_hostdefiner.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: csi.ibm.com/v1
+kind: HostDefiner
+metadata:
+  name: ibm-block-csi
+  namespace: default
+  labels:
+    app.kubernetes.io/name: ibm-block-csi
+    app.kubernetes.io/instance: ibm-block-csi
+    app.kubernetes.io/managed-by: ibm-block-csi-operator
+    release: v1.10.0
+spec:
+  hostDefiner:
+    #prefix:
+    #connectivityType:
```

Comment:
```suggestion
    # prefix:
    # connectivityType:
```

Reply:
why? I wanted that when the user will remove the `#` the fields will be aligned with the other fields, do you think moving the `#` one char backward will be still good?

Reply:
on second thought:
```suggestion
#    prefix:
#    connectivityType:
```

Reply:
done

Reply:
note there are 4 spaces, not 3 (so that it would be aligned after deleting the `#`)